src/shared/converters/image.js

```diff
@@ -40,45 +39,67 @@ export default {
           eventAction="open content media"
         >
           <a>
-            <Image content key={attachmentId} id={attachmentId} />
+            <Image isContent key={attachmentId} id={attachmentId} />
           </a>
         </Link>
       );
     }
 
-    let src;
+    const alt = attributes.alt || null;
 
-    // Get src attribute from different cases or assign an empty string.
+    let src = null;
+
+    // Get src attribute from different cases.
     if (attributes.src && typeof attributes.src === 'string') {
-      ({ src } = attributes);
+      if (attributes.src.startsWith('http')) src = he.decode(attributes.src);
+      else src = he.decode(`${settings.generalSite.url}${attributes.src}`);
```

Comment:
If I'm not mistaken, there are 4 possible values for `src`:
1. `<img src="http://www.myweb.com/location/of/image.png"` absolute
1. `<img src="/location/of/image.png"` local, but from the root
1. `<img src="location/of/image.png"` local, but from the same folder your html is in
1. `<img src="data:image/png;base64,iVBORw0KGgo...` embeded

Right now you're only contemplating 1 and 2. We should contemplate at least 4.